its/src/test/java/com/sonar/it/jenkins/SonarPluginTestSuite.java

```diff
@@ -83,8 +81,11 @@ public static void setUpJenkins() {
       .build());
     DEFAULT_QUALITY_GATE_NAME = getDefaultQualityGateName();
 
-    EARLIEST_JENKINS_SUPPORTED_MS_BUILD_VERSION = SCANNER_VERSION_PROVIDER
-        .getEarliestSupportedVersion("sonar-scanner-msbuild");
+    ScannerAvailableVersionsProvider scannerProvider = new ScannerAvailableVersionsProvider();
+    ScannerAvailableVersionsProvider.ScannerAvailableVersions scannerVersions = scannerProvider.getScannerAvailableVersions("sonar-scanner-msbuild");
+
+    EARLIEST_JENKINS_INSTALLABLE_MSBUILD_VERSION = scannerVersions.getOldest();
+    LATEST_JENKINS_MSBUILD_VERSION = scannerVersions.getLatest();
```

Comment:
We should streamline variable names. What about:
* `OLDEST_INSTALLABLE_MSBUILD_VERSION`
* `LATEST_INSTALLABLE_MSBUILD_VERSION`